module3/homework/advanced_calculator/test.cpp

```diff
@@ -88,15 +88,15 @@ TEST(advancedCalculatorTest, ShouldCalculateSqrt) {
     double result = 0;
 
     ASSERT_EQ(process("225 $3", &result), ErrorCode::OK);
-    EXPECT_TRUE(cmp(result, 8.77915e-08));
+    EXPECT_TRUE(cmp(result, 6.0822));
```

Comment:
Why not use EXPECT_DOUBLE_EQ or EXPECT_NEAR? https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#floating-point-macros?